### ISSUE TITLE
docs.rs page linked in the readme doesn't exist " rustpython-0.1.2 is not a library."

### ISSUE THREAD

**Issue by konstin** [RustPython/RustPython#4784]
The readme currently says

> You can read the [online documentation](https://docs.rs/rustpython) for the
> latest release, or the [user guide](https://rustpython.github.io/docs/).

Opening https://docs.rs/rustpython however then just says "rustpython-0.1.2 is not a library."

**Comment by DimitrisJim** [comment]
Yeah, that's kinda unfortunate and I'm unsure what is the history behind it. As a temporary solution, you can build the docs locally:

```
cargo doc --no-deps -p rustpython-common -p rustpython-compiler-core -p rustpython-compiler -p rustpython-codegen -p rustpython-parser -p rustpython-vm -p rustpython-stdlib -p rustpython-jit -p rustpython-derive -p rustpython
```

you can, of course, specify a subset of the packages to build if you'd like to see a specific one, i.e:

```
cargo doc --no-deps -p rustpython-compiler
```

**Comment by konstin** [comment]
yep, i'm currently doing `cargo doc --open` in our downstream project, mostly opened this because the readme contained a dead link
